tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -87,6 +88,11 @@ const (
 	redisInstallName = "redis-release"
 )
 
+const (
```

Comment:
the list of all constants for this file is above. I'd add to that (separating w/ a blank line).